OpenRA.Mods.Common/Traits/Sound/SoundOnDamageTransition.cs

```diff
@@ -8,33 +8,45 @@
  */
 #endregion
 
+using OpenRA.Support;
 using OpenRA.Traits;
 
 namespace OpenRA.Mods.Common.Traits
 {
 	public class SoundOnDamageTransitionInfo : ITraitInfo
 	{
-		public readonly string DamagedSound;
-		public readonly string DestroyedSound;
+		[Desc("Play a random sound from this list when damaged.")]
+		public readonly string[] DamagedSounds = { };
+
+		[Desc("Play a random sound from this list when destroyed.")]
+		public readonly string[] DestroyedSounds = { };
 
 		public object Create(ActorInitializer init) { return new SoundOnDamageTransition(this); }
 	}
 
 	public class SoundOnDamageTransition : INotifyDamageStateChanged
 	{
-		readonly SoundOnDamageTransitionInfo info;
+		public readonly SoundOnDamageTransitionInfo Info;
```

Comment:
Because someone may want access to it (as we do with other traits) and this isn't hurting anything.